fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -1225,6 +1225,17 @@ private void getHelperNodes(String[] args) throws AnalysisException {
         LOG.info("get helper nodes: {}", helperNodes);
     }
 
+    private void transferToMasterWithOldType(FrontendNodeType oldType) {
+        feType = FrontendNodeType.MASTER;
```

Comment:
just use MASTER to init, I think it would be unsafe.

Reply:
I set the type to MASTER after all the pre works ready.